src/styles/TopNavStyles.ts

```diff
@@ -0,0 +1,50 @@
+import styled from 'styled-components';
+import AppBar from '@material-ui/core/AppBar';
+import IconButton from '@material-ui/core/IconButton';
+import Toolbar from '@material-ui/core/Toolbar';
+import Typography from '@material-ui/core/Typography';
+
+export const Root = styled.div`
+    flex-grow: 1;
+`;
+
+export const Bar = styled(AppBar)`
+    position: sticky;
+    background: transparent;
+    box-shadow: none;
+`;
+
+export const ToolBar = styled(Toolbar)`
+    && { min-height: 128px;
+    align-items: flex-start;
+    padding-top: 8px;
```

Comment:
don't think padding should be necessary here -- there's already some built-in padding in the MUI component :)